protolint.yaml

```diff
@@ -5,4 +5,5 @@ lint:
       - ENUM_FIELD_NAMES_ZERO_VALUE_END_WITH
```

Comment:
Running `protolint` (and by extension, generate protocols script) on my machine updated the `.proto` files (but no ton CI). Added this rule to ensure consistency of protocols in case the script is run in another env like mine (Mac or Python version 3.9) which is different to CI's (Ubuntu Py 3.8).